framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -99,6 +99,7 @@ qx.Class.define("qx.ui.table.model.Filtered",
      * @param the_needle {String} String to search
      * @param the_haystack {Array} Array, which should be searched
      * @return {Boolean} whether the search string was found.
+     * @deprecated {6.0} Use qx.lang.Array.contains instead
```

Comment:
I don't know the origin of that but this is just like qx.lang.Array.contains (and less efficient). I kept just because it's not private